clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -1065,6 +1067,14 @@ message FilterResult {
     map<string, CollectorResult> nestedCollectorResults = 2;
 }
 
+message LoggingHits {
+    // number of hits to log. For example, totalHits = 20 and hitsCount = 50 means that 50 documents will be logged
+    // while 20 documents will be returned in the Search Response
+    int32 hitsCount = 1;
```

Comment:
I don't think I see this value being used anywhere. Also, what happens when `hitsCount` is less than `topHits`? Is there a default value?

Reply:
I added here to have LoggingHits already defined but I will use hitsCount in an upcoming PR. In a second thought, it is better to include hitsCount in the following PR with tests.